firebase-functions/src/androidTest/java/com/google/firebase/functions/CallTest.java

```diff
@@ -90,7 +90,8 @@ public void testToken() throws InterruptedException, ExecutionException {
             () -> {
               HttpsCallableContext context = new HttpsCallableContext("token", null);
               return Tasks.forResult(context);
-            });
+            },
+            null);
```

Comment:
would it be possible to get a named variable for these nulls? Its unclear what exactly they mean for the test?

Something like "NO_CALLBACK" or something.

Reply:
Good idea.